.pre-commit-config.yaml

```diff
@@ -43,8 +43,6 @@ repos:
       - id: check-added-large-files
       - id: check-ast
       - id: check-builtin-literals
-        args:
```

Comment:
@bandish-shah any concerns with allowing `dict` literals? the mmdet configs are hell without them.

Reply:
@mvpatel2000 Hi mihir! This line requires a review from someone on composer-team-admins it's allowing the use of `dict` literals. mmdetection configs are in this form, i feel that it hurts readability to convert them. I'm also not sure what we have against `dict`?